frontend/app/src/components/defi/loan/loans/makerdao/ManageWatchers.vue

```diff
@@ -24,7 +24,7 @@ const watchers = computed(() => {
   const { identifier } = get(vault);
   return get(loanWatchers).filter(watcher => {
     const watcherArgs = watcher.args;
-    return watcherArgs.vault_id.includes(identifier);
+    return watcherArgs.vaultId.includes(identifier);
```

Comment:
We don't have a zod parser for the watcher data do we?

Can you make sure to also add a parser?

Reply:
okay I will add it